### ISSUE TITLE
Seeding rank no longer updated for stopped torrents

### ISSUE THREAD

**Issue by goedl** [BiglySoftware/BiglyBT#2218]
* OS and version
Win10
* BiglyBT Version Number
2.8.0.0

Hope it's OK to ask a question here:
TLDR: Is it intentional that since 2.7. stopped torrents no loger get a "seeding rank"?
Could this eventually come back in the future?

With the introduction of "Light Seeding" in 2.7. I have some problems keeping my work flow efficient. I want/need to set many torrents to "Stopped" and because of the adjusted seeding rank I previously still could recognize if a leecher would need a seed by floating this stopped torrent to the top of my (few) active torrents sorted by seeding rank. In this cases I want to individualy decide to start the torrent. This no longer works even when Light Seeding is switched off because the seeding rank apparently is no longer adjusted for stopped torrents.

I can't switch on Light-Seeding since apparently my mainly used privat tracker detects those torrents as active and since it only allows a limited number of active torrents I'm receiving error messages that I have too many active seeds.
My current workaround is to sort by Peers but this is a bit cumbersome and I would much prefer to have the behaviour as is was < 2.7.

**Comment by parg** [comment]
A PT complaining about too much seeding? These guys are never happy.

There is a config option to limit the number of "light seeds" - not sure how strictly it is enforced but you could play with that if interested.

If you join the Beta program (Help menu option) and update to 2801_B14 I have fiddled with the seeding rank column - see if that is doing what you want.

**Comment by parg** [comment]
Hmm, OK, well that's showing the rank but not sorting correctly by it. I'll look further!

**Comment by parg** [comment]
Try B15

**Comment by goedl** [comment]
> Try B15

Thanks a lot. Seems to work as expected.
